ray-on-gke/tpu/kuberay-tpu-webhook/main.go

```diff
@@ -635,6 +607,41 @@ func init() {
 func main() {
 	flag.Parse()
 
+	// use in-cluster config if kubeConfig path is not passed as a flag
+	if KubeConfigPath == "" {
+		config, err := rest.InClusterConfig()
+		if err != nil {
+			panic(err)
+		}
+		client = kubernetes.NewForConfigOrDie(config)
+	} else {
+		config, err := clientcmd.BuildConfigFromFlags("", KubeConfigPath)
+		if err != nil {
+			panic(err)
+		}
+		client = kubernetes.NewForConfigOrDie(config)
+	}
+
+	// instantiate PodInformer for Ray worker pods in the GKE cluster
+	tweakListOptionsFunc := func(options *metav1.ListOptions) {
+		options.LabelSelector = "ray.io/node-type=worker"
+	}
+	factory := informers.NewFilteredSharedInformerFactory(client, 5*time.Minute, metav1.NamespaceAll, tweakListOptionsFunc)
+	podInformer = factory.Core().V1().Pods()
```

Comment:
podInformer = factory.Core().V1().Pods().Informer()

Reply:
or 

```
podLister = factory.Core().V1().Pods().Lister()
```

Use type v1.PodLister for the variable: https://github.com/kubernetes/client-go/blob/master/informers/core/v1/pod.go#L39

Reply:
[f4e3a6a](https://github.com/GoogleCloudPlatform/ai-on-gke/pull/723/commits/f4e3a6ac4d4c70f806d813f3e2775dd30b6657fc)